### PR TITLE
Update documentation for Cozy V3

### PR DESCRIPTION
**Pull request by clochix**

### CODE REVIEW

**Comment by nono** [comment]
@jsilvestre @aenario ping

**Comment by jsilvestre** [comment]
Sorry, didn't see it until today, I don't look at my github emails often :-/

Gotta review it asap!

File: source/v2.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
+---
+title: Cozy V3 Developer Documentation
```
Comment (aenario) [inline source/v2.md]: s/v3/v2

**Review by aenario — APPROVED** [review]

File: source/v2.md (hunk 1 above)
Comment (clochix) [reply source/v2.md]: thanks, fixed

**Comment by aenario** [comment]
@jsilvestre good to merge for me.

**Review by jsilvestre — APPROVED** [review]
It's good, thank you @clochix, we'll see what to improve based on feedback!

**Comment by jsilvestre** [comment]
I merge and deploy it.

**Pull request merged**